python/GangaLHCb/Lib/Splitters/GaussSplitter.py

```diff
@@ -39,39 +42,59 @@ def _create_subjob(self, job, inputdata):
         return j
 
     def split(self, job):
+        """
+            Method to do the splitting work
+            Args:
+                job (Job): master job to be used as a template to split subjobs
+        """
+
+        # NB Not sure if we can ever _not_ import the Gauss app from the GPI
+        from Ganga.GPI import Gauss
```

Comment:
Your solution in a44814872b0b2ab77964a8c36c3f81c4c653f074 is correct. The problem I have is not with this PR but rather with the magical way we are generating these applications in the LHCb plugin.